sdk/core/azure-core/samples/README.md

```diff
@@ -0,0 +1,7 @@
+# Azure Core Samples
+
+test_example_sync.py - samples of how to create a sync pipeline
+
+test_example_async.py - samples of how to create an async pipeline
+
+test_example_sansio.py - samples of how to config policies
```

Comment:
would suggest applying formatting to filenames (emph or pre) will make things a little easier to distinguish visually